### PR TITLE
Resteasy 1426 readme updates

### PR DESCRIPTION
**Pull request by rsearls**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,41 +1,41 @@
...
 The project page can be found at http://resteasy.jboss.org
 
 ## JAX-RS
-RESTEasy is a JBoss project that provides various frameworks to help you build RESTful Web Services and RESTful Java applications. It is a portable implementation of the JAX-RS specification. JAX-RS is a new JCP specification that provides a Java API for RESTful Web Services over the HTTP protocol.
+RESTEasy is a JBoss project that provides various frameworks to help you build RESTful Web Services and RESTful Java applications. It is a portable implementation of the JAX-RS specification. JAX-RS is a JCP specification that provides a Java API for RESTful Web Services over the HTTP protocol.
 
 Read the entire specification [here](https://jax-rs-spec.java.net/)
 
-The JAX-RS code is in the jaxrs directory.
+The RESTEasy, JAX-RS code is in the github repository, (https://github.com/resteasy/Resteasy.git).
```
Comment (asoldano) [inline README.md]: I propose: "The RESTEasy code is in the github repository, (https://github.com/resteasy/Resteasy.git)"

**Comment by asoldano** [comment]
If you can, please rebase this PR squashing all the changes into a single commit, otherwise I'll cherry-pick the last one instead of merging the PR.

**Comment by rsearls** [comment]
Update complete.  Squashing was done.

----- Original Message -----

**Comment by marekkopecky** [comment]
What do you think about removing README.html file? It contains only information about documentation, but this information is also in README.md ...

**Comment by rsearls** [comment]
I don't have any objection.  I would suggest asking on the resteasy-dev alias before doing it.

**Comment by asoldano** [comment]
+1, just remove that html file, it has no added value afaics.

**Comment by rsearls** [comment]
File removed and squashed

**Comment by asoldano** [comment]
Merged (after having squashed everything in a single commit), thanks

**Pull request closed without merging**